tests/test_build.nf.test.snap

```diff
@@ -0,0 +1,47 @@
+{
+    "test_build": {
```

Comment:
There are some processes to which I do not see the files here. For example: `arriba_download`. Aren't these running?

Reply:
Good catch, this is due to the --all false specified in the test_buid.nf.test. We can't run it with --all in github actions, due to the size of the output it generates, so only defaults are run.

Reply:
You can pass each argument independently such as: `--arriba true` `--fusioncatcher true` `--starfusion_build true` etc. You would then avoid the processes that are too big to run when passing the `--all`.

```
    if (params.arriba || params.all) {
        ARRIBA_DOWNLOAD()
    }
```

Reply:
Ah, I understand what you mean, I explored using those parameters, but the issue is not with `arriba` but with `STAR_GENOMEGENERATE` which gets run whenever any of all, starfusion, arriba or starindex are true:
```
    if (params.starindex || params.all || params.starfusion || params.arriba) {
        STAR_GENOMEGENERATE( ENSEMBL_DOWNLOAD.out.primary_assembly, ENSEMBL_DOWNLOAD.out.gtf )
    }
```

Reply:
got it, let's keep it as it is then.